src/Filter/ClassFilter.php

```diff
@@ -39,10 +39,6 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
 
         $operator = $this->getOperator((int) $data['type']);
```

Comment:
This parameter and the line above looks a bit strange compared to the other filters, can we equalize?